keentools/blender_independent_packages/pykeentools_loader/config.py

```diff
@@ -23,6 +23,9 @@
 
 import bpy
 
+from ...addon_config import Config
```

Comment:
I don't like this backward dependency. Maybe move ` if Config.mock_update_for_testing_flag:` to a parent module